modules/gitops-logging-into-keycloak.adoc

```diff
@@ -0,0 +1,65 @@
+[id="gitops-logging-into-keycloak_{context}"]
+= Logging into Keycloak
+
+By logging in to the Keycloak console after configuring the Argo CD instance, you can manage identities or roles and define the permissions assigned to the various roles.
+
```

Comment:
For this procedure it is essential that you have removed the default configuration of Dex and added the Keycloak configuration. Hence it must be captured as a prerequisite.
Your Argo CD CR must be configured to use the Keycloak SSO provider.